lib/devices/ios/ios.js

```diff
@@ -1016,18 +1016,37 @@ IOS.getDeviceStringFromOpts = function (opts) {
       'iPad Simulator (8.3 Simulator)': 'iPad 2 (8.3 Simulator)',
       'iPad Simulator (8.4 Simulator)': 'iPad 2 (8.4 Simulator)',
       'iPad Simulator (7.1 Simulator)': 'iPad 2 (7.1 Simulator)',
-      'iPad Simulator (9.0)': 'iPad 2 (9.0)',
       'iPhone Simulator (8.4 Simulator)': 'iPhone 6 (8.4 Simulator)',
       'iPhone Simulator (8.3 Simulator)': 'iPhone 6 (8.3 Simulator)',
       'iPhone Simulator (8.2 Simulator)': 'iPhone 6 (8.2 Simulator)',
       'iPhone Simulator (8.1 Simulator)': 'iPhone 6 (8.1 Simulator)',
       'iPhone Simulator (8.0 Simulator)': 'iPhone 6 (8.0 Simulator)',
-      'iPhone Simulator (9.0)': 'iPhone 6 (9.0)',
       'iPhone Simulator (7.1 Simulator)': 'iPhone 5s (7.1 Simulator)'
     };
-    if (CONFIG_FIX[iosDeviceString]) {
+    // For xcode major version 7
+    var CONFIX_FIX__XCODE_7 = {
+      'iPad Simulator (8.1)': 'iPad 2 (8.1)',
+      'iPad Simulator (8.2)': 'iPad 2 (8.2)',
+      'iPad Simulator (8.3)': 'iPad 2 (8.3)',
+      'iPad Simulator (8.4)': 'iPad 2 (8.4)',
+      // Fixing ambiguous device name for ios9.0 and above;#5619
+      'iPad Simulator (9.0)': 'iPad 2 (9.0) [',
+      'iPad Simulator (9.1)': 'iPad 2 (9.1) [',
+      'iPhone Simulator (8.1)': 'iPhone 6 (8.1)',
+      'iPhone Simulator (8.2)': 'iPhone 6 (8.2)',
+      'iPhone Simulator (8.3)': 'iPhone 6 (8.3)',
+      'iPhone Simulator (8.4)': 'iPhone 6 (8.4)',
+      // Fixing ambiguous device name for ios9.0 and above;#5619
+      'iPhone Simulator (9.0)': 'iPhone 6 (9.0) [',
+      'iPhone Simulator (9.1)': 'iPhone 6 (9.1) ['
+    };
+    var confixFix = CONFIG_FIX;
```

Comment:
`var confixFix = xcodeMajorVer === 7 ? CONFIX_FIX__XCODE_7 : CONFIG_FIX;`

Also, `configFix` instead of `confixFix`.